From a25f50554ba61e92d6dd0863c227b043ffb338e6 Mon Sep 17 00:00:00 2001
From: Sunil Goutham <sgoutham@marvell.com>
Date: Wed, 21 Aug 2019 22:06:18 +0530
Subject: [PATCH 327/767] octeontx2-af: Fix compilation issue

commit 13d8c3dd95622989a2f9499b64dbf2dfa3189f44 from
git@git.assembla.com:cavium/WindRiver.linux.git

Fix compilation issue introduced by patch
'commit e833c99ef4dd ("octeontx2-af: Transmit packets during SMQ flush")'

Change-Id: Ice1473e779d04263577e73d5e83e2d925642cdc9
Signed-off-by: Sunil Goutham <sgoutham@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/14606
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
Reviewed-on: https://sj1git1.cavium.com/14930
Tested-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 drivers/net/ethernet/marvell/octeontx2/af/rvu_fixes.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/drivers/net/ethernet/marvell/octeontx2/af/rvu_fixes.c b/drivers/net/ethernet/marvell/octeontx2/af/rvu_fixes.c
index e9e9c07335c2..5dac149dd1e0 100644
--- a/drivers/net/ethernet/marvell/octeontx2/af/rvu_fixes.c
+++ b/drivers/net/ethernet/marvell/octeontx2/af/rvu_fixes.c
@@ -999,6 +999,11 @@ bool is_parse_nibble_config_valid(struct rvu *rvu,
 	return true;
 }
 
+void __weak otx2smqvf_xmit(void)
+{
+	/* Nothing to do */
+}
+
 void rvu_smqvf_xmit(struct rvu *rvu)
 {
 	if (is_rvu_95xx_A0(rvu) || is_rvu_96xx_A0(rvu)) {
-- 
2.31.1

